coops-ui/tests/unit/ARDate.spec.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint promise/param-names: 0, prefer-promise-reject-errors: 0 */
```

Comment:
Severin: Overriding ESLint warnings/errors should only be used as a last resort. Is it not possible to fix the flagged warnings/errors?

Cameron: Yes, not sure how this got in here. Will remove! Thanks for the catch.